### PR TITLE
Add Catogram, MX Player/Pro, Snapseed, Solid Explorer, DriveDroid + more

### PR DESCRIPTION
**Pull request by ninopiamonte**
You're welcome and no worries @patrykmichalik. Btw, I still don't know how to properly align the drawable icons I just convert them online so if you see any errors, please correct/optimize them and I'll just happily apply/commit changes. Thanks!

#42 #21 #20

### CODE REVIEW

**Comment by ninopiamonte** [comment]
@patrykmichalik should I update? I've attached the Solid Explorer outline version if you want to modify it. current & suggestion:

Screen Shot 2021-11-13 at 12 00 54 AM

[solid_explorer_outline.zip](https://github.com/LawnchairLauncher/lawnicons/files/7528877/solid_explorer_outline.zip)

**Comment by patrykmichalik** [comment]
I feel like the existing Solid Explorer icon better resembles the official design, so I’d suggest keeping this icon as is. On the other hand, I’d say XDA icon on the right is an improvement over the existing one. It may deviate from the official app icon, but it should be more legible.

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
I would suggest nudging the XDA icon down so that it looks optically centered: [xda-icon-suggestion.zip](https://github.com/LawnchairLauncher/lawnicons/files/7532420/xda-icon-suggestion.zip). Because this PR deletes lines from `xda.svg`, I’m not able to use GitHub’s suggestion feature here.

**Comment by ninopiamonte** [comment]
> I would suggest nudging the XDA icon down so that it looks optically centered: [`xda-icon-suggestion.zip`](https://github.com/LawnchairLauncher/lawnicons/files/7532420/xda-icon-suggestion.zip). Because this PR deletes lines from `xda.svg`, I’m not able to use GitHub’s suggestion feature here.

Got it, thanks!

**Comment by ninopiamonte** [comment]
Should close #73

**Comment by ninopiamonte** [comment]
I forgot to optically align the Stellio icon. Will modify it tomorrow lol

**Comment by paphonb** [comment]
For truecaller please link to `@drawable/dialer` instead of creating a new icon. Otherwise looks fine.

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -90,5 +91,19 @@
...
+    <icon drawable="@drawable/catogram" package="ua.itaysonlab.messenger" name="Catogram" />
+    <icon drawable="@drawable/mx_player" package="com.mxtech.videoplayer.pro" name="MX Player" />
+    <icon drawable="@drawable/mx_player" package="com.mxtech.videoplayer.ad" name="MX Player" />
+    <icon drawable="@drawable/snapseed" package="com.niksoftware.snapseed" name="Snapseed" />
+    <icon drawable="@drawable/solid_explorer" package="pl.solidexplorer2" name="Solid Explorer" />
+    <icon drawable="@drawable/drivedroid" package="com.softwarebakery.drivedroid" name="DriveDroid" />
     <icon drawable="@drawable/facebook" package="com.facebook.lite" name="Facebook" />
+    <icon drawable="@drawable/bromite" package="org.bromite.bromite" name="Bromite" />
+    <icon drawable="@drawable/stellio" package="io.stellio.music" name="Stellio" />
+    <icon drawable="@drawable/truecaller" package="com.truecaller" name="Truecaller" />
```
Comment (patrykmichalik) [inline app/src/main/res/xml/grayscale_icon_map.xml:103]:
```suggestion
    <icon drawable="@drawable/dialer" package="com.truecaller" name="Dialer" />
```

File: app/src/main/res/drawable/sweech.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000000"
+        android:pathData="M21.92,2.08 L21.92,2 L4.801,2.001 L2.08,4.722 L2.08,8.69 L8.695,15.305 L13.513,20.123 L7.894,16.107 L2.08,21.92 L2.08,22 L19.199,21.999 L21.92,19.278 L21.92,15.31 L15.305,8.695 L10.487,3.877 L16.106,7.893 L21.92,2.08 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/sweech.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="M3.072,5.4498l0,3.5712l5.9535,5.9535l4.3362,4.3362l-5.0571,-3.6144l-5.2326,5.2317l0,0.072l15.4071,-0.0009l2.4489,-2.4489l0,-3.5712l-5.9535,-5.9535l-4.3362,-4.3362l5.0571,3.6144l5.2326,-5.2317l0,-0.072l-15.4071,0.0009z"/>
</vector>
```

File: svgs/sweech.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <polygon points="21.92 2.08 21.92 2 4.801 2.001 2.08 4.722 2.08 8.69 8.695 15.305 13.513 20.123 7.894 16.107 2.08 21.92 2.08 22 19.199 21.999 21.92 19.278 21.92 15.31 15.305 8.695 10.487 3.877 16.106 7.893 21.92 2.08"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/sweech.svg:3]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="sweech.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs7" />
  <sodipodi:namedview
     id="namedview5"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="32.375"
     inkscape:cx="11.984556"
     inkscape:cy="12"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <polygon
     points="2.08,4.722 2.08,8.69 8.695,15.305 13.513,20.123 7.894,16.107 2.08,21.92 2.08,22 19.199,21.999 21.92,19.278 21.92,15.31 15.305,8.695 10.487,3.877 16.106,7.893 21.92,2.08 21.92,2 4.801,2.001 "
     id="polygon2"
     transform="matrix(0.9,0,0,0.9,1.2,1.2)" />
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Looking good! I suggest updating `grayscale_icon_map.xml` as per @paphonb’s suggestion and making the Sweech icon a bit smaller. The icon is more or less a square, and squares are optically larger than, for example, circles.

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
Thanks, @ninopiamonte!

**Pull request merged**